modules/core/src/main/java/org/apache/fluo/core/impl/VisibilityCache.java

```diff
@@ -48,9 +48,9 @@ public int weigh(Bytes key, ColumnVisibility vis) {
 
   VisibilityCache() {
```

Comment:
You can modify this constructor to take a `FluoConfiguration`.  The viscache constructor is only called in `SharedResources` which can be changed to do `visCache = new VisibilityCache(env.getConfiguration())`

Reply:
Ok @keith-turner this should all be taken care of with this next commit